torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -1591,18 +1575,27 @@ std::vector<Val*> Index::getGlobalProducerStridedIndices(
       continue;
     }
 
-    TORCH_INTERNAL_ASSERT(
+    Val* root_ind = nullptr;
+    auto override_it = override_index.find(root_dom[i]);
```

Comment:
I was thinking about passing the optional map to `getTensorIndexFromIdGraph` to provide an initial ID-to-index map. That would be more consistent if we would want to allow the same initial map in consumer indexing.

That said, I think this is good enough for now given that the whole indexing code would be redesigned. 

Pinging @csarofeen

Reply:
Agreed